header.tpl

```diff
@@ -0,0 +1,16 @@
+ * This file is part of the NMPolicy project
```

Comment:
I do not think we need this line... Placing the one on line 15 here should do the job.